src/Debug/ShutdownHandler.php

```diff
@@ -31,13 +27,7 @@ public static function register($debug = true)
         self::$errorLevels = error_reporting();
 
         if (PHP_SAPI === 'cli') {
-            $consoleHandler = function (\Exception $e) {
-                $app = new Application('Bolt CLI', Version::VERSION);
-                $output = new ConsoleOutput(OutputInterface::VERBOSITY_DEBUG);
-                $app->renderException($e, $output);
-                ob_clean();
-            };
```

Comment:
FWIW this was to stop a lot of mess on the console when exceptions landed … This would be a regression?